dashboard/app/controllers/rubrics_controller.rb

```diff
@@ -1,10 +1,43 @@
 class RubricsController < ApplicationController
+  include Rails.application.routes.url_helpers
```

Comment:
Can you add `before_action :require_levelbuilder_mode_or_test_env` here? (There are many examples of using this but [here's](https://github.com/code-dot-org/code-dot-org/blob/55677a769a54248329f56db1b8d6664737f93641/dashboard/app/controllers/data_docs_controller.rb#L2) one). This restricts these routes to only levelbuilder or test so that they can't be used in production.